protos/flyteidl/core/workflow.proto

```diff
@@ -68,6 +68,35 @@ message WorkflowNode {
     }
 }
 
+// SignalCondition represents a dependency on an existing signal.
```

Comment:
Can you clarify what you mean by an existing signal?

Reply:
I was probably caught up in the flyteadmin side; where a signal does not technically "exist", meaning it can be provided, until propeller processes the gatenode that contains the signal condition. I'll go ahead and remove this, it can be confusing.